packages/run-protocol/test/reserve/setup.js

```diff
@@ -61,27 +61,27 @@ const setupReserveBootstrap = async (
 
 /**
  * @typedef {{
- * reserveCreatorFacet: import('../../src/reserve/collateralReserve').CollateralReserveCreatorFacet,
- * reservePublicFacet: import('../../src/reserve/collateralReserve').CollateralReservePublicFacet,
- * instance: Instance,
+ *   reserveCreatorFacet: import('../../src/reserve/collateralReserve').CollateralReserveCreatorFacet;
+ *   reservePublicFacet: import('../../src/reserve/collateralReserve').CollateralReservePublicFacet;
+ *   instance: Instance;
  * }} ReserveKit
  */
 
 /**
  * NOTE: called separately by each test so contracts don't interfere
  *
- * @param {{ committeeName: string, committeeSize: number}} electorateTerms
- * @param {ManualTimer | undefined=} timer
+ * @param {{ committeeName: string; committeeSize: number }} electorateTerms
+ * @param {ManualTimer | undefined} [timer]
  * @returns {{
- *     zoe: ZoeService,
- *     feeMintAccess: FeeMintAccess,
- *     installation: Installation,
- *     committeeCreator: ElectorateCreatorFacet,
- *     electorateInstance: Instance,
- *     governor: any,
- *     reserve: ReserveKit,
- *     invitationAmount: Amount,
- *     space: any,
+ *   zoe: ZoeService;
```

Comment:
semicolons. That’s interesting. Is that consistent with TS?

Reply:
I believe TSC prefers semicolons